aws-lambda/src/main.rs

```diff
@@ -13,8 +15,17 @@ mod runtime;
 
 #[tokio::main]
 async fn main() -> Result<(), Error> {
+    let _ = dotenv();
+
+    let trace: tracing::Level = std::env::var("LOG_LEVEL")
```

Comment:
```suggestion
    let trace: tracing::Level = std::env::var("TC_LOG_LEVEL")
```
https://tailcall.run/docs/guides/logging/